src/sweeper/sweepers/duplicates.py

```diff
@@ -21,13 +22,19 @@ def sweep(self):
         '''
         report = {'title': 'Duplicate Test', 'feature_class': self.table_name, 'issues': []}
         digests = set([])
+        is_table = False
 
         truncate_shape_precision = re.compile(r'(\d+\.\d{2})(\d+)')
 
         with arcpy.EnvManager(workspace=self.workspace):
             description = arcpy.da.Describe(self.table_name)
+            print(f'Working on Duplicates for: {self.table_name}')
+            if description['dataType'] == 'Table':
```

Comment:
**suggestion**: casefold or lowercase this type of string comparisons.

Reply:
Done.